ckanext/data_qld_theme/templates/datarequests/show.html

```diff
@@ -18,7 +18,7 @@
   {% endif %}
 
   {% if h.check_access('open_datarequest', {'id':datarequest_id }) and c.datarequest.closed %}
-    {% link_for _('Re-open'), named_route='open_datarequest', id=datarequest_id, class_='btn btn-success', icon='unlock' %}
+    {% link_for _('Re-open'), controller='ckanext.data_qld.controller:DataQldUI', action='open_datarequest', id=datarequest_id, class_='btn btn-success', icon='unlock' %}
```

Comment:
Can't we just use the new plugin straight away (and decom this plugin?)

Reply:
Unfortunately not. The merge was done in an attempt to simplify the move to CKAN 2.9, and thus it's intertwined with those changes.